src/VisualStudio/IntegrationTest/TestUtilities/InProcess/DialogHelpers.cs

```diff
@@ -19,6 +34,72 @@ public static AutomationElement FindDialog(int visualStudioHWnd, string dialogAu
                 delay: TimeSpan.FromMilliseconds(250));
         }
 
+        /// <summary>
+        /// Selects a specific item in a combo box.
+        /// Note that combo box is found using its Automation ID, but the item is identified by name.
+        /// </summary>
+        public static void SelectComboBoxItem(int visualStudioHWnd, string dialogAutomationName, string comboBoxAutomationName, string itemText)
+        {
+            var dialogAutomationElement = GetOpenDialog(visualStudioHWnd, dialogAutomationName);
+
+            var comboBoxAutomationElement = dialogAutomationElement.FindDescendantByAutomationId(comboBoxAutomationName);
+            comboBoxAutomationElement.Expand();
+
+            var comboBoxItemAutomationElement = comboBoxAutomationElement.FindDescendantByName(itemText);
+            comboBoxItemAutomationElement.Select();
+
+            comboBoxAutomationElement.Collapse();
+        }
+
+        /// <summary>
+        /// Selects a specific radio button.
+        /// </summary>
+        public static void SelectRadioButton(int visualStudioHWnd, string dialogAutomationName, string radioButtonAutomationName)
+        {
+            var dialogAutomationElement = GetOpenDialog(visualStudioHWnd, dialogAutomationName);
+
+            var radioButton = dialogAutomationElement.FindDescendantByAutomationId(radioButtonAutomationName);
+            radioButton.Select();
+        }
+
+        /// <summary>
+        /// Sets the value of the specified element in the dialog.
+        /// Used for setting the values of things like combo boxes and text fields.
+        /// </summary>
+        public static void SetElementValue(int visualStudioHWnd, string dialogAutomationName, string elementAutomationName, string value)
+        {
+            var dialogAutomationElement = GetOpenDialog(visualStudioHWnd, dialogAutomationName);
+
+            var control = dialogAutomationElement.FindDescendantByAutomationId(elementAutomationName);
+            control.SetValue(value);
+        }
+
+        /// <summary>
+        /// Presses the specified button.
+        /// The button is identified using its automation ID; see <see cref="PressButtonWithName(int, string, string)"/>
+        /// for the equivalent method that finds the button by name.
+        /// </summary>
+        public static void PressButton(int visualStudioHWnd, string dialogAutomationName, string buttonAutomationName)
```

Comment:
Change "buttonAutomationName" to "buttonAutomationId" (and I think you probably mean the same for dialogAutomationId, right?)